src/content/docs/journey-performance/slo-tutorial.mdx

```diff
@@ -0,0 +1,296 @@
+---
+title: 'Create front-end performance benchmarks with service levels'
+metaDescription: 'Learn how to create and define performance benchmarks with servic levels.'
+---
+
+import sonicSliUiOverview from 'images/sonic_screenshot-full_sli-ui-overview.png'
+
+## Objectives
+
+By the end of this tutorial, you will:
```

Comment:
I feel like the Objectives and Intro sections being separate gives a kind of disjointed feeling to the beginning. I think with these tutorials we should consider starting with something to really grab users and bring them in based on the task based nature of the doc, purely from a VSU perspective for potential customers.